includes/definitions/cnpilotr.yaml

```diff
@@ -0,0 +1,12 @@
+os: cnpilotr
+text: Cambium cnPilot Router
+type: network
+icon: cambium
+mib_dir: cambium
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+group: cambium
+discovery:
+    -
+        sysDescr_regex:
```

Comment:
For completeness I would add sysObjectId: .1.3.6.1.4.1.17713.22 here.

Reply:
This device's sysObjectId only returns  .1.3.6.1.4. which is basically useless otherwise i would have included it.